js/models/contact_model.js

```diff
@@ -9,22 +9,24 @@ angular.module('contactsApp')
 			addressBookId: addressBook.displayName,
 
 			uid: function(value) {
+				var model = this;
 				if (angular.isDefined(value)) {
 					// setter
-					return this.setProperty('uid', { value: value });
+					return model.setProperty('uid', { value: value });
 				} else {
 					// getter
-					return this.getProperty('uid').value;
+					return model.getProperty('uid').value;
 				}
 			},
 
 			fullName: function(value) {
+				var model = this;
```

Comment:
and this line doesn't:
https://github.com/owncloud/contacts/pull/269/files#diff-25fe07d2d32ea4ff3c47532c18614c20R26

Either way no reason to prevent the merge.